src/utils/configuration.ts

```diff
@@ -7,9 +7,16 @@ export interface SnapshotConfiguration {
   'min-height'?: number,
 }
 
+export interface StaticSiteSnapshotConfiguration {
```

Comment:
Wonder if this should be its own file to match the setup of the other config options